src/js/components/Calendar/__tests__/Calendar-test.tsx

```diff
@@ -916,3 +791,104 @@ describe('Calendar Keyboard events', () => {
     expect(asFragment()).toMatchSnapshot();
   });
 });
+
+describe('Calendar Keyboard events', () => {
```

Comment:
Is there a reason we had to move the test order? Or just preference?

Reply:
I wanted the 'heading font size' test to not be grouped into the keyboard events describe block and instead be grouped in the main calendar test describe block